CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing
```

Comment:
The top of this file closely follows the format of Tailwind's own [contributing doc](https://github.com/tailwindlabs/tailwindcss/blob/master/.github/CONTRIBUTING.md).